### PR TITLE
[CONTEXT-263] Add mixpanel to Discovery~~

### PR DESCRIPTION
**Pull request by ghost**
@anderscognite  pls add me to mixpanel

### CODE REVIEW

File: src/containers/ModelList.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```
Comment (anderscognite) [inline src/containers/ModelList.tsx]: Why is this change in mixpanel PR?

**Review by anderscognite — CHANGES_REQUESTED** [review]
Just one unexpected change in this PR (or a couple unrelated changes I guess). Can you split these into different PR's? I think that every line in a PR should be reflected in the PR title / description.

File: src/containers/ModelList.tsx (hunk 1 above)
Comment (dlcognite) [reply src/containers/ModelList.tsx]: man bad rebase since this file was moved in master... fixed.

File: src/utils/metrics.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import * as mixpanelConfig from 'mixpanel-browser';
+import { sdk } from '../index';
+
+const mixpanel = mixpanelConfig.init(
+  'fb25742efb56d116b736515a0ad5f6ef',
```
Comment (anderscognite) [inline src/utils/metrics.ts]: Can we extract this into a variable called `MIXPANELTOKEN` or something? Just feels a bit cleaner.

**Review by anderscognite — APPROVED** [review]

**Pull request merged**